apps/server/src/modules/deletion/uc/deletion-request.uc.ts

```diff
@@ -17,15 +17,11 @@ import { DeletionRequest } from '../domain/deletion-request.do';
 import { DeletionOperationModel } from '../domain/types/deletion-operation-model.enum';
 import { DeletionStatusModel } from '../domain/types/deletion-status-model.enum';
 import { DeletionLog } from '../domain/deletion-log.do';
-import {
-	DeletionRequestProps,
-	DeletionRequestLog,
-	DeletionLogStatistic,
-	DeletionRequestCreateAnswer,
-} from './interface/interfaces';
+import { DeletionLogStatistic } from './interface/interfaces';
 import { DeletionLogStatisticBuilder } from './builder/deletion-log-statistic.builder';
-import { DeletionRequestLogBuilder } from './builder/deletion-request-log.builder';
+import { DeletionRequestLogResponseBuilder } from './builder/deletion-request-log-response.builder';
```

Comment:
Please shorten this import (as well as the one below) - export these files via `builder`'s dir barrel file (`index.ts`).

Reply:
done